src/main/java/com/googlecode/lanterna/gui2/AbstractListBox.java

```diff
@@ -297,6 +303,44 @@ public synchronized V getSelectedItem() {
         }
     }
 
+	/**
+	 * Removes the item with the selected index from the list.
+	 * 
+	 * @param index the Index of the item to delete
+	 * @return Itself
+	 * @throws IndexOutOfBoundsException If the index is less than zero or
+	 *             equals/greater than the number of items in the list
+	 */
+	public synchronized T removeItem(int index) {
+		if (selectedIndex > -1 && !items.isEmpty()) {
```

Comment:
In retrospect, we can probably get by without the first check against selectedIndex. If `index is < 0` then the `IndexOutOfBounds` exception will just bubble up from the remove on the underlying list object. I think the original point of checking `selectedIndex` here was to ensure that the list box was the active component (although you wouldn't necessarily need the list box to be the active component to do a remove). So I'd say we can either replace `selectedIndex` with `index` or just remove that check altogether.